subprojects/plugins/src/main/java/org/gradle/api/plugins/JavaBasePlugin.java

```diff
@@ -352,6 +354,15 @@ private void configureBuildDependents(Project project) {
         });
     }
 
+    private void configureArchiveDefaults(Project project) {
+        // TODO: Gradle 8.1+: Deprecate `getLibsDirectory` in BasePluginExtension and move it to `JavaPluginExtension`
```

Comment:
Alternatively, we could get rid of this distinction and put all the jars in one place.

Reply:
In the same place as the dists? Not a bad idea. Why is this even configurable? I can't imagine why a user cares where the jar really exists, unless they want to find it in their build dir -- in which case it only really matters that its in a consistent location.

Reply:
I think people definitely do care that they can find where output jars are created - even if these are "intermediate" operations and not the final product of an ear/war build.  Inspecting the contents of a jar is a common debugging step.  I think we should call this out in the upgrade guide if it changes.

Reply:
Yeah thats fair. I've definitely opened up jars in that dir a fair number of times. So while I think its important that we have the jars in an accessible location in the build dir, I'm still curious how many people are actually going and changing that directory. 

Agree though that if the default ever changes we should call that out.